src/domain/accounts.ts

```diff
@@ -0,0 +1,181 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the Apache License, Version 2.0 (the 'License') and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Sridhar Voruganti <sridhar.voruganti@modusbox.com>
+
+ --------------
+ ******/
+
+import { Util as HapiUtil } from '@hapi/hapi'
+import Logger from '@mojaloop/central-services-logger'
+
+import {
+  Enum,
+  Util,
+  FspEndpointTypesEnum,
+  RestMethodsEnum
+} from '@mojaloop/central-services-shared'
+// eslint is complaining about these imports. not sure why.
+// eslint-disable-next-line import/no-unresolved
+import Config from '~/shared/config'
+import { inspect } from 'util'
+import {
+  APIErrorObject,
+  FSPIOPError,
+  ReformatFSPIOPError
+} from '@mojaloop/central-services-error-handling'
+// eslint-disable-next-line import/no-unresolved
+import { finishChildSpan } from '~/shared/util'
+// eslint-disable-next-line import/no-unresolved
+import * as types from '~/interface/types'
+
+/**
+ * @function forwardAccountsIdRequestError
+ * @description Generic function to handle sending `PUT .../accounts/{ID}/error` back to the FSPIOP-Source
+ * @param {string} path Callback endpoint path
+ * @param {HapiUtil.Dictionary<string>} headers Headers object of the request
+ * @param {string} userId the ID of the /accounts/{ID} resource
+ * @param {APIErrorObject} error Error details
+ * @param {object} span optional request span
+ * @throws {FSPIOPError} Will throw an error if no endpoint to forward the accounts requests is
+ *  found, if there are network errors or if there is a bad response
+ * @returns {Promise<void>}
+ */
+export async function forwardAccountsIdRequestError (
+  path: string,
+  headers: HapiUtil.Dictionary<string>,
+  userId: string,
+  error: APIErrorObject,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
instead of any use unknown

Reply:
If we change span type to unknown,  lint complains at span?.getChild, then also need to typecast it to any